src/nvim/editor/arbmark.h

```diff
@@ -0,0 +1,27 @@
+#ifndef NVIM_ARBMARK_H
+#define NVIM_ARBMARK_H
+
+/* #include "nvim/vim.h" */
+/* #include "nvim/lib/kvec.h" */
+
+#define ARBMARK_MAXLEN 24
+#define pos_cmp(a, b) (_pos_cmp((a).fmark.mark, (b).fmark.mark))
+
+typedef struct {
+  fmark_T fmark;
+  fmark_T *next;
+  fmark_T *prev;
+} arbmark_T;
```

Comment:
CamelCase, `_T` suffix is legacy. Given the rename I guess this should be ExtendedMark.